### PR TITLE
Certbot howto

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,3 +93,119 @@ DEPLOYMENT=stg make zuul-secrets
...
+$ cd /tmp/haxxx/
+$ python3 serve-acme-challenge.py /.well-known/acme-challenge/xyz abc
+```
+Values of `<abc>` and `<xyz>` are generated by cert-bot in the first terminal session.
+
+Return to first terminal and Press Enter to Continue.
+
+And that's it! We got the certs now, let's get them from the pod to this git repo:
+```
+$ oc rsh pod/get-them-certs-1-p6xhm cat /tmp/live/prod.packit.dev/cert.pem | dos2unix > secrets/prod/cert.pem
```
Comment (rpitonak) [inline README.md]: no idea what `dos2unix` is

Comment (jpopelka) [reply README.md]: For some reason the result of `oc rsh pod/get-them-certs-1-p6xhm cat /tmp/live/prod.packit.dev/cert.pem` has `^M` characters at the end of lines.
`dos2unix fixes those`, see https://unix.stackexchange.com/questions/32001/what-is-m-and-how-do-i-get-rid-of-it
 Ok, I can add a comment there.

**Comment by jpopelka** [comment]
Thanks Rado,

I have one more - see the latest commit - I realized that we actually don't [deploy](https://github.com/packit-service/deployment/blob/master/openshift/secret-packit-secrets.yml.j2) `cert.pem` & `chain.pem` - so, do we need them at all?

**Comment by rpitonak** [comment]
> I have one more - see the latest commit - I realized that we actually don't [deploy](https://github.com/packit-service/deployment/blob/master/openshift/secret-packit-secrets.yml.j2) `cert.pem` & `chain.pem` - so, do we need them at all?

Yes, that is actually a good point. Obviously we do not need them so +1 for removing.

File: README.md [hunk 2]
```diff
@@ -93,3 +93,120 @@ DEPLOYMENT=stg make zuul-secrets
...
+$ python3 serve-acme-challenge.py /.well-known/acme-challenge/xyz abc
+```
+Values of `<abc>` and `<xyz>` are generated by cert-bot in the first terminal session.
+
+Return to first terminal and Press Enter to Continue.
+
+And that's it! We got the certs now, let's get them from the pod to this git repo:
+```
+$ oc rsh pod/get-them-certs-1-abcde cat /tmp/live/prod.packit.dev/cert.pem | dos2unix > secrets/prod/cert.pem
+$ oc rsh pod/get-them-certs-1-abcde cat /tmp/live/prod.packit.dev/chain.pem | dos2unix > secrets/prod/chain.pem
```
Comment (TomasTomecek) [inline README.md]: these 2 are not needed then, right?

Comment (jpopelka) [reply README.md]: They are most likely not, but I left it there after some thinking.
Ok, I'll remove them and add some comment about them.

Comment (jpopelka) [reply README.md]: #13

**Pull request merged**